docs/diagrams/StorageSaveSequence.puml

```diff
@@ -0,0 +1,49 @@
+@startuml
+'https://plantuml.com/sequence-diagram
+
+Box
+    participant Main
+    participant ":ExitCommand" as ExitCommand
+    participant ":Storage" as Storage
+    participant ":PersonList" as PersonList
+    participant PlanITarium.txt
+end box
+
+->> Main: execute `bye`
+Main ->> ExitCommand: exit()
+ExitCommand ->> Storage: saveData(:Family)
+activate Storage
+loop for all 3 family groups
+Storage->>PersonList: getPersonList()
+activate PersonList
+PersonList-->>Storage: :PersonList
+deactivate PersonList
+    loop until all items
+        Storage->>PersonList: getSaveName()
+        activate Storage
```

Comment:
Possibly extra activation as the extra "box" is used to show a method of an object calling another of its own methods. Believe that all the extra boxes below can be removed if they are not methods of the Storage class